superset-frontend/cypress-base/cypress/integration/explore/visualizations/table.test.js

```diff
@@ -103,7 +103,8 @@ describe('Visualization > Table', () => {
   it('Test table with columns and row limit', () => {
     const formData = {
       ...VIZ_DEFAULTS,
-      query_mode: 'raw',
+      // should still work when query_mode is not-set/invalid
+      query_mode: undefined,
```

Comment:
should we keep the old test case _and_ add the new one?

Reply:
There is another test below that sets `query_mode` with a value. Can make it more explicit though.